include/bluetooth/mesh/gen_lvl.h

```diff
@@ -50,8 +50,6 @@ extern "C" {
 struct bt_mesh_lvl_set {
 	/** New level. */
 	int16_t lvl;
-	/** Whether this is a new action. */
-	bool new_transaction;
```

Comment:
@michal-narajowski are these needed for PTS testing the gen_lvl_cli?

Reply:
No, they are not needed for PTS testing.